mlir/lib/Dialect/MIOpen/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -1627,7 +1642,18 @@ struct GridwiseGemmV2RewritePattern
                               GemmABlockCopyThreadSliceLengths_GemmM};
     }
 
-    uint32_t blockwiseVectorDimA = matrix_a_source_vector_read_dim;
+    auto dimToIndex = [](GemmDimension dim) {
```

Comment:
I think this is spelled `static_cast<uint32_t>(dim)`
and if it isn't, I'd rather see this as a `static` function and not a lambda

Reply:
Yep, agreed that a static_cast is cleaner way to do it. Done in f5e59f4.